code/modules/food_and_drink/plants.dm

```diff
@@ -488,11 +491,12 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
 /obj/item/reagent_containers/food/snacks/plant/melon/george
 	name = "rainbow melon"
 	crop_prefix = "rainbow "
-	desc = "Sometime in the year 2472 these melons were required to have their name legally changed to protect the not-so-innocent. Also for tax evasion reasons."
+	desc = "Sometime in the year 2472 these melons were required to have their name legally changed to protect the not-so-innocent. \
```

Comment:
i know it wasnt you who wrote this but the game is set in the year 2053. 2472 is... quite a lot after that. perhaps change the year.

Reply:
ok!